Misc/ACKS

```diff
@@ -2104,5 +2104,6 @@ Jelle Zijlstra
 Gennadiy Zlobin
 Doug Zongker
 Peter Åstrand
+Lucas Esposito
```

Comment:
Find the `E` names to add yourself to (line immediately below here says to use alphabetical order, which for English alphabets doesn't have to be "rough").

Reply:
Fixed! My bad.

In a quick look I though the list had no apparent order... but it was just that the list started with first name even though it's sorted by last name. I didn't notice at first